tests/network/hotplug_bridge.go

```diff
@@ -231,7 +255,24 @@ var _ = Describe(SIG("bridge nic-hotplug", func() {
 	})
 }))
 
-var _ = Describe(SIG("bridge nic-hotunplug", func() {
+var _ = Describe(SIG("bridge nic-hotunplug", Serial, func() {
+	BeforeEach(func() {
+		virtClient := kubevirt.Client()
+		originalKv := libkubevirt.GetCurrentKv(virtClient)
+		updateStrategy := &v1.KubeVirtWorkloadUpdateStrategy{
+			WorkloadUpdateMethods: []v1.WorkloadUpdateMethod{v1.WorkloadUpdateMethodLiveMigrate},
+		}
+		rolloutStrategy := pointer.P(v1.VMRolloutStrategyLiveUpdate)
+		patchWorkloadUpdateMethodAndRolloutStrategy(originalKv.Name, virtClient, updateStrategy, rolloutStrategy)
+
+		currentKv := libkubevirt.GetCurrentKv(virtClient)
+		config.WaitForConfigToBePropagatedToComponent(
+			"kubevirt.io=virt-controller",
+			currentKv.ResourceVersion,
+			config.ExpectResourceVersionToBeLessEqualThanConfigVersion,
+			time.Minute)
+	})
```

Comment:
this blocks repeats right ?
please consider adding a function
it also repeats on next commit of this PR `tests/network/hotplug_sriov.go`

it also might repeat on other files (`tests/hotplug/instancetype.go` unless not exactly the same)
anyhow if it is messy that both will use the same, we can just start at least with the repetitions in this PR

Reply:
> this blocks repeats right ? please consider adding a function it also repeats on next commit of this PR `tests/network/hotplug_sriov.go`
> 
> it also might repeat on other files (`tests/hotplug/instancetype.go` unless not exactly the same) anyhow if it is messy that both will use the same, we can just start at least with the repetitions in this PR

This code was adjusted from the CPU hotplug tests.
It makes sure the LiveUpdate Rollout strategy is set prior to running the tests.

I think we could consider it for a follow-up.

Reply:
well you are the maintainer, but i think this is an example it scream repetitions
if the helper function is problematic as it cross package then fine, but what about simple one in the same file / package ?